doc/ref/configuration/minion.rst

```diff
@@ -1485,6 +1485,53 @@ this can be set to ``True``.
 
     always_verify_signature: True
 
+.. conf_minion:: cmd_blacklist_glob
+
+``cmd_blacklist_glob``
+----------------------
+
```

Comment:
Both of these new options should probably have a `.. versionadded:: Carbon` tag.